cmake/BLTMacros.cmake

```diff
@@ -1401,3 +1401,26 @@ macro(blt_export_tpl_targets)
         endif()
     endforeach()
 endmacro()
+
+##------------------------------------------------------------------------------
+## blt_convert_to_system_includes(TARGET <target>)
+##
+## Converts existing interface includes to system interface includes.
+##------------------------------------------------------------------------------
+macro(blt_convert_to_system_includes)
+    set(options)
+    set(singleValuedArgs TARGET)
+    set(multiValuedArgs)
+
+    ## parse the arguments to the macro
+    cmake_parse_arguments(arg
+         "${options}" "${singleValuedArgs}" "${multiValuedArgs}" ${ARGN})
+
+    if(NOT DEFINED arg_TARGET)
+       message(FATAL_ERROR "TARGET is a required parameter for the blt_convert_to_system_includes macro.")
+    endif()
+
+    get_target_property(_include_dirs ${arg_TARGET} INTERFACE_INCLUDE_DIRECTORIES)
+    target_include_directories(${arg_TARGET} SYSTEM INTERFACE ${_include_dirs}))
```

Comment:
This will duplicate the paths in the `INTERFACE_INCLUDE_DIRECTORIES`... sorry.

Reply:
OK, I've gone back to the original implementation.